test/cypress/integration/agreements/agreement.cy.js

```diff
@@ -42,7 +42,75 @@ describe('Agreement detail page', () => {
     cy.setEncryptedCookies(userExternalId)
   })
 
-  it('should display agreement detail page and allow cancelling agreement', () => {
+  it('should display agreement detail page', () => {
+    cy.task('setupStubs', [
+      ...userAndGatewayAccountStubs(),
+      agreementStubs.getLedgerAgreementSuccess({
+        service_id: serviceExternalId,
+        live: false,
+        gatewayAccountId,
+        external_id: 'a-valid-agreement-id'
+      }),
+      transactionsStub
+    ])
+
+    cy.visit('/test/service/service-id/account/gateway-account-id/agreements/a-valid-agreement-id')
+
+    cy.get('h1').contains('Agreement detail')
+
+    cy.get('.govuk-heading-l').should('have.text', 'Agreement detail')
+
+    cy.get('[data-cy=agreement-detail]').get('h2').contains('Payment instrument')
```

Comment:
This assertion is ever so slightly confusing as we're also asserting on the "Payment instrument" title further down when we have assertions for the payment instrument section. It makes it seem like the assertions below this might be looking at the payment instrument section.

Maybe we can remove this one?

Reply:
👍